modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/objectbuilder/ParameterGroupObjectBuilder.java

```diff
@@ -83,15 +99,31 @@ public T build(ResolverSetResult result) throws MuleException {
     }
   }
 
-  private T doBuild(Predicate<String> hasParameter, Function<String, Object> parameters, ValueResolvingContext context)
+  private T doBuild(Predicate<String> hasParameter, Function<String, Object> parameters, ValueResolvingContext context,
+                    CursorComponentDecoratorFactory componentDecoratorFactory)
       throws MuleException {
     T object = createInstance(prototypeClass);
 
     for (FieldElement field : groupDescriptorFields) {
       String name = field.getName();
       if (hasParameter.test(name)) {
-        Object resolvedValue = resolveValue(new StaticValueResolver<>(parameters.apply(name)), context);
-        Object value = context == null || context.resolveCursors() ? resolveCursor(resolvedValue) : resolvedValue;
+        final boolean isContent = field.isAnnotatedWith(Content.class)
+            || field.isAnnotatedWith(org.mule.sdk.api.annotation.param.Content.class);
+        Object resolvedValue = resolveValue(new StaticValueResolver<>(parameters
```

Comment:
same comment as before. Not sure if appropriate to do this now of wait for that refactor till the input and output decorations are merged.